### DISCUSSION TITLE
Hexo.io Page Unresponsive Error

### DISCUSSION THREAD

**Discussion by mdaehn** [hexojs/hexo#5267] [Question / Answer]
Not sure were to post this, but your hexo.io site keeps getting a "Page Unresponsive" error. I am new to using Hexo, and it is a little concerning. I also was looking at some of your themes and it seem to take me to a page that said I had a virus, although I couldn't reproduce it a second time around. I was using Chrome 115.0.5790.98 on a Macbook Pro 2023 M2 Max.

**Comment by stevenjoezhang** [comment]
You can use alternative networks to access the website and check if the issue still persists, such as using your mobile's cellular network. It's possible that your connection has been hijacked by the service provider or affected by DNS contamination.

Websites related to certain themes might lack maintenance and their domain names could be registered by others after expiration, and then used for malicious purposes. We will be removing these expired links in the future.

**Comment by uiolee** [comment]
Please provide URL or screenshot etc.
